.rubocop_todo.yml

```diff
@@ -0,0 +1,90 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2017-10-31 15:01:47 +0100 using RuboCop version 0.51.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 2
+Lint/AmbiguousRegexpLiteral:
+  Exclude:
+    - 'build-caasp-iso.rb'
+
+# Offense count: 2
+Lint/ImplicitStringConcatenation:
+  Exclude:
+    - 'build-caasp-iso.rb'
+
+# Offense count: 1
+Lint/RescueWithoutErrorClass:
+  Exclude:
+    - 'build-caasp-iso.rb'
+
+# Offense count: 3
+Lint/ShadowingOuterLocalVariable:
+  Exclude:
+    - 'build-caasp-iso.rb'
+
+# Offense count: 1
+Lint/UselessAssignment:
+  Exclude:
+    - 'build-caasp-iso.rb'
+
+# Offense count: 3
+Metrics/AbcSize:
+  Max: 31
+
+# Offense count: 1
+# Configuration parameters: CountComments.
+Metrics/ClassLength:
+  Max: 117
+
+# Offense count: 16
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
+# URISchemes: http, https
+Metrics/LineLength:
+  Max: 140
+
+# Offense count: 7
+# Configuration parameters: CountComments.
+Metrics/MethodLength:
+  Max: 21
+
+# Offense count: 3
+Style/Documentation:
+  Exclude:
+    - 'spec/**/*'
+    - 'test/**/*'
+    - 'build-caasp-iso.rb'
+
+# Offense count: 1
+# Configuration parameters: MinBodyLength.
+Style/GuardClause:
+  Exclude:
+    - 'build-caasp-iso.rb'
+
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, AllowInnerSlashes.
+# SupportedStyles: slashes, percent_r, mixed
+Style/RegexpLiteral:
+  Exclude:
+    - 'build-caasp-iso.rb'
+
+# Offense count: 48
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles, ConsistentQuotesInMultiline.
+# SupportedStyles: single_quotes, double_quotes
+Style/StringLiterals:
+  Exclude:
+    - 'Gemfile'
+    - 'build-caasp-iso.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: single_quotes, double_quotes
+Style/StringLiteralsInInterpolation:
+  Exclude:
+    - 'build-caasp-iso.rb'
```

Comment:
Why the need to disable cops here? The ideal thing would be to avoid disabling cops, in any case, if it's necessary you can do it inline (in the source code), something like:

```
# rubocop:disable Style/StringLiteralsInInterpolation
def some_offender_method
end
# rubocop:enable Style/StringLiteralsInInterpolation
```

This way is obvious in which part of the code we are disabling a cop and for what reason.

Reply:
@ereslibre  i didn't disable cops there.

This rubocop_todo.yml file is auto-generated by
``` rubocop --auto-gen-config```

here is another upstream https://github.com/bbatsov/rubocop/blob/master/.rubocop_todo.yml#L84

Reply:
Oh! sorry, will read about that then. Thank you!